bindep.txt

```diff
@@ -20,6 +20,7 @@ libffi-devel      [platform:rpm]
 openssl-devel     [platform:rpm]
 podman            [platform:rpm]
 python3-devel     [platform:rpm !platform:rhel-7 !platform:centos-7]
+python3-libvirt   [platform:rpm]
```

Comment:
so we may need to use another base image for the container, or add some repository in order to get that package.